### PR TITLE
[Geometries] Removes current geometry creation files and scripts

### PR DESCRIPTION
**Pull request by Kongkille**
Related to #3545 

This PR removes all geometry scripts that are used to generate world.json and zonegeometries:

- Changes to the dockerfile, such that docker build doesn't try to generate the map
- Removes all geo scripts except for 'generate-zone-bounding-boxes'
- Removes all third party maps

This means that after this merge, we cannot regenerate world.json and zonegeometries using the current method, however nothing else should break.

When we merge #3545 we can start generating new worlds again!

### CODE REVIEW

File: web/Dockerfile [hunk 1]
```diff
@@ -7,13 +7,8 @@ ADD web/package.json ./package.json
...
 RUN yarn install --frozen-lockfile
 
-# Generate map
-RUN mkdir -p public/dist/
-ADD web/geo/third_party_maps ./geo/third_party_maps
-ADD web/geo/ ./geo/
-ADD web/src/world.json ./src/world.json
-RUN bash geo/topogen.sh
 
+ADD web/src/world.json ./src/world.json
```
[the remark this answers is not in the document]
Comment (pierresegonne) [reply web/Dockerfile:11]: Can you test it?

[the remark this answers is not in the document]
Comment (Kongkille) [reply web/Dockerfile:11]: I tried to build with and without with no apparent destruction, but I don't understand docker all that well. @skovhus would you know?

[the remark this answers is not in the document]
Comment (skovhus) [reply web/Dockerfile:11]: It should be redundant as line 22 should take care of this. But let us leave it here for now.

**Review by skovhus — APPROVED** [review]

**Comment by madsnedergaard** [comment]
Shouldn't this have waited with merging until the other PR is done? 🤔

**Comment by Kongkille** [comment]
The purpose of it was to be able to be done regardless, as I thought zonegeometries could just be 'left' in public/dist as a static file, but it seems it only appears after it's been build.

**Pull request merged**